lib/data/static_data.rb

```diff
@@ -9,15 +9,57 @@ class StaticData
   DROPLET_SIZE = 's-1vcpu-1gb'
   SSH_KEY_ID = ''
 
+  DEFAULT_USER = 'root'
+
+PATHS_LIST = [
+        {dir: '.s3',             file: 'key'},
+        {dir: '.s3',             file: 'private_key'},
+        {dir: '.palladium',      file: 'token'},
+        {dir: '.documentserver', file: 'documentserver_jwt'}
+        ].freeze
+
+  SPEC_FILES = %w[check_open_docx_by_screen_spec.rb
+                  check_open_epub_by_screen_spec.rb
+                  check_open_fb2_by_screen_spec.rb
+                  check_open_odp_by_screen_spec.rb
+                  check_open_odt_by_screen_spec.rb
+                  check_open_pptx_by_screen_spec.rb
+                  check_open_xlsx_by_screen_spec.rb
+                  check_open_ods_by_screen_spec.rb
+                  check_open_html_by_screen_spec.rb].freeze
+
+  DOCKERFILE = "/#{StaticData::DEFAULT_USER}/convert-service-testing/Dockerfile"
+  ENV = "/#{StaticData::DEFAULT_USER}/convert-service-testing/.env"
+
+  GIT_CLONE_PROJECT = 'git clone https://github.com/ONLYOFFICE-QA/convert-service-testing.git'
+
   def self.get_project_name
       return ENV['PROJECT_NAME'] if ENV['PROJECT_NAME']
 
-      File.read("#{Dir.home}/.do/project_name").delete("\n")
+      File.read("#{Dir.home}/.do/project_name").rstrip
   end
 
   def self.get_ssh_key_id
       return ENV['SSH_KEY_ID'] if ENV['SSH_KEY_ID']
 
-      File.read("#{Dir.home}/.do/ssh_key_id").delete("\n")
+      File.read("#{Dir.home}/.do/ssh_key_id").rstrip
+  end
+
+  def self.get_palladium_token
```

Comment:
I don't think any of this methods need `get` in there title

`def self.palladium_token` is clear enought